packages/appstore/src/components/wallet/wallet-card.tsx

```diff
@@ -182,7 +185,7 @@ const WalletCard = ({ active, balance, currency, dark, demo, disabled, faded, si
                 ) : (
                     <div className={classNames('wallet-card__logo', 'wallet-card__logo--placeholder')} />
                 )}
-                {size !== 'small' && (
+                {!['xsmall', 'small'].includes(size || '') && (
```

Comment:
@iman-fs `['xsmall', 'small'].includes(size || '')` has been used twice, Can extract it into a helper function.

Reply:
I agree. done